app/components/Settings/Preview/sql-table.jsx

```diff
@@ -113,46 +115,29 @@ export default class SQLTable extends React.Component {
 
     render() {
         return (
-            <div>
-                <div
-                    className={'sqltable-click-overlay'}
-
-                    style={{
-                        float: 'left',
-                        minWidth: '100%',
-                        minHeight: '32px',
-                        position: 'absolute',
-                        zIndex: 2
-                    }}
-
-                    onClick={() => this.refs.reactDataGrid.onToggleFilter()}
-                />
+            <div className={'sqltable'}>
+                <code>
+                    <small>
+                        <a onClick={() => this.refs.reactDataGrid.onToggleFilter()}>
+                            Toggle Row Filter
+                        </a>
+                    </small>
+                </code>
                 <ReactDataGrid
                     ref={'reactDataGrid'}
 
                     minWidth={740}
-                    minHeight={200}
+                    minHeight={300}
```

Comment:
Minor thing.  Do these variables belong in const?

Reply:
No, I wouldn't. My personal criteria for defining a `const` is:
- if the constant is used only once or twice, I only define a `const` if it documents the meaning; e.g. in `minHeight={300}`, I don't define a constant, because it's clear that 300 is the minimum height passed to ReactDataGrid.
- if the constant is use twice or more times, then I define a constant, because is easier to update `const MY_CONST = 300;` in one place than 300 in multiple places.